lib/util.js

```diff
@@ -454,6 +454,30 @@ const util = {
     ]
     util.run('ninja', ninjaOpts, options)
   },
+ 
+  runCoverage: (options = config.defaultOptions) => {
+    console.log('running coverage for ' + config.buildTarget + '...')
+
+    if (process.platform === 'win32') util.updateOmahaMidlFiles()
+    if (process.platform === 'linux') util.prepareWidevineCdmBuild()
+
+    const args = util.buildArgsToString(config.buildArgs())
+    // provide arg for COVERAGE_OUT_DIR
+    util.run('gn', ['gen',
+      'out/brave/coverage', // make an arg - i.e. build with COVERAGE_OUT_DIR as 'out/COVERAGE_OUT_DIR/coverage'
```

Comment:
we don't want a separate output dir for code coverage, what are you trying to do here?